deepmd/pt/model/atomic_model/linear_atomic_model.py

```diff
@@ -158,6 +158,14 @@ def get_model_rcuts(self) -> list[float]:
     def get_sel(self) -> list[int]:
         return [max([model.get_nsel() for model in self.models])]
 
+    def set_dataid(self, data_idx):
+        """
+        Set the data identification of this atomic model by the given data_idx,
+        typically concatenated with the output of the descriptor and fed into the fitting net.
+        """
+        for model in self.models:
+            model.set_dataid(data_idx)
```

Comment:
check if model has `set_dataid` method? seems pairtab model will raise `NotImplementedError`

Reply:
Maybe that's more clear to rasie `NotImplementedError` inside pairtab model, rather than specifically check here?